src/TypeDefinition.js

```diff
@@ -229,6 +229,7 @@ export type NavigationStackScreenOptions = NavigationScreenOptions & {
   headerTintColor?: string,
   headerLeft?: React.Element<*>,
   headerBackTitle?: string,
+  headerBackImage?: any,
```

Comment:
Could you point me to correct type? I tried to use Image.propTypes.source but got some errors...